components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -0,0 +1,295 @@
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+import { connectAccount } from '../../lib/api';
+
+class AcceptContributionsMyself extends React.Component {
+  static propTypes = {
+    collective: PropTypes.object,
+    intl: PropTypes.object.isRequired,
+    router: PropTypes.object,
+    LoggedInUser: PropTypes.object.isRequired,
+    addBankAccount: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: null,
+    };
+
+    this.messages = defineMessages({
+      header: {
+        id: 'acceptContributions.picker.header',
+        defaultMessage: 'Accept financial contributions',
+      },
+      howDoesItWork: {
+        id: 'paymentMethods.manual.HowDoesItWork',
+        defaultMessage: 'How does it work?',
+      },
+      howDoesItWorkDetails: {
+        id: 'acceptContributions.HowDoesItWork.details',
+        defaultMessage:
+          'Financial contributors will be able to choose "Bank transfer" as a payment method. Instructions to make the transfer, which you define, will be emailed to them, along with a unique order ID. Once you receive the money, you can mark the corresponding pending order as paid and the funds will be credited to the Collective\'s balance.',
+      },
+      definePaymentInstructions: {
+        id: 'acceptContributions.definePaymentInstructions',
+        defaultMessage: 'Define payment instructions',
+      },
+      definePaymentInstructionsDetails: {
+        id: 'acceptContributions.definePaymentInstructionsDetails',
+        defaultMessage:
+          'Include any details contributors will need to send you money, such as: account name and number; IBAN, Swift, or routing codes; bank name and address, etc. The amount and order ID will be automatically included.',
+      },
+      cancel: {
+        id: 'actions.cancel',
+        defaultMessage: 'Cancel',
+      },
+      save: {
+        id: 'save',
+        defaultMessage: 'Save',
+      },
+      addBankAccount: {
+        id: 'acceptContributions.addBankAccount',
+        defaultMessage: 'Add bank account',
+      },
+      howAreYouAcceptingContributions: {
+        id: 'acceptContributions.howAreYouAcceptingContributions',
+        defaultMessage: 'How are you accepting contributions?',
+      },
+    });
+  }
+
+  connectStripe = () => {
+    const { collective } = this.props;
+    const service = 'stripe';
+
+    connectAccount(collective.id, service)
+      .then(json => {
+        return (window.location.href = json.redirectUrl);
+      })
+      .catch(err => {
+        console.error(`>>> /api/connected-accounts/${service} error`, err);
+      });
+  };
+
+  submitBankAccountInformation = async bankAccountInfo => {
+    // set state to loading
+    this.setState({ loading: true });
+
+    // prepare objects
+    const account = {
+      legacyId: this.props.LoggedInUser.CollectiveId,
+    };
+
+    const value = {
+      name: 'Bank ACCOUNT',
```

Comment:
Strange capitalization

Reply:
Apologies, I was copying what I thought I saw on @znarf's screen when he showed the host 'add bank account' instructions. I can change it.